sys/include/net/ng_sixlowpan/frag.h

```diff
@@ -42,10 +42,14 @@ extern "C" {
 #define NG_SIXLOWPAN_FRAG_SIZE_MASK     (0x07ff)    /**< mask for datagram size */
 
 /**
- * @brief   General and 1st 6LoWPAN fragmentation header
+ * @brief   1st 6LoWPAN fragmentation header
  *
- * @see <a href="https://tools.ietf.org/html/rfc4944#section-5.1">
- *          RFC 4944, section 5.1
+ * @note    Before the implementation determines if the header is a 1st or a
+ *          subsequent 6LoWPAN fragmentation header, this it is casted to this
+ *          data type.
```

Comment:
Ok, but then I will add a line to explain, what we mean by "general fragmentation header", because after looking at the RFC this remained completely unclear to me.